scripts/test_twitter_ingestion.py

```
@@ -0,0 +1,157 @@
+            data = json.loads(json_str)
+            
+            # Take only first few records for testing
+            if isinstance(data, list) and len(data) > max_records:
+                data = data[:max_records]
+            
+            return data
+        else:
+            print(f"⚠️  Could not extract JSON from {file_name}")
+            return None
+            
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)